gsy_framework/sim_results/aggregate_results.py

```diff
@@ -154,7 +154,13 @@ def generate(self):
         next_time = start_time + self.resolution - self.simulation_slot_length
         timeslots_to_aggregate = []
         for timeslot in sorted_market_timeslots:
-            if timeslot >= next_time:
+            if timeslot > next_time:
```

Comment:
In order to avoid code duplication, I would maintain the >= and add an extra clause inside the block:
```
if timeslot > next_time:
    timeslots_to_aggregate = [timeslot]
else:
    timeslots_to_aggregate = []
```
since as far as I see this is the only difference in the 2 blocks.

Reply:
The order of things is also different. I preferred the code to be easier to read rather than perfectly written here. But still, I don't like how this is written either.

Reply:
I see, then it is fine by me. I do not see an immediate or simple way to fix this unfortunately